README.md

```diff
@@ -429,21 +431,6 @@ sudo apt install libavdevice-dev # If you want to read from video devices such a
 sudo apt install pandoc  # If you want to recreate the man page
 ```
 
-#### Get dependencies on macOS
```

Comment:
I'd keep the instructions to build it from source.

Reply:
For the structure should we have:
### Install pre-built package
#### Debian/Ubuntu 
_(once accepted)_
`apt ....` 
#### macOS 
`brew....` 
### Build from source 
#### Get dependencies on Debian/Ubuntu
#### Get dependencies on macOS
#### Get repo 
_Get repo after we install git_
#### Compile timg
#### Install